docs/sections/user_guide/yaml/tags.rst

```diff
@@ -0,0 +1,103 @@
+.. _defining_YAML_tags:
+
+UW YAML tags
+============
+
+Tags are used to denote the type of a YAML node when converting to a Python object. Standard YAML tags are defined at http://yaml.org/type/index.html.
+
+Tags may be implicit:
+
+.. code-block:: yaml
+
+   boolean: true
+   integer: 3
+   float: 3.14
+
+Or explicit:
+
+.. code-block:: yaml
+
+   boolean: !!bool "true"
+   integer: !!int "3"
+   float: !!float "3.14"
+
+Additionally, UW defines the following tags to support use cases not covered by standard tags:
+
+``!float``
+^^^^^^^^^^
+
+Converts the tagged node to a Python ``float`` value.
+
+* Given input.yaml
+  .. code-block:: yaml
+
+    f2: !float "{{ 3.141 + 2.718 }}"
+
+  .. code-block:: text
+     % uw config realize --input-file input.yaml --output-format yaml
+     f2: 5.859
+
+
+``!int``
+^^^^^^^^
+
+Converts the tagged node to a Python ``int`` value.
+
+
+* Given input.yaml
+  .. code-block:: yaml
+
+   values:
+    f1: 3
+    f2: 11
+    f3: !int "{{ (f1 + f2) * 10 }}"
```

Comment:
```suggestion
  .. code-block:: yaml

     values:
       f1: 3
       f2: 11
       f3: !int "{{ (f1 + f2) * 10 }}"
```

Reply:
Typo there along with the broken formatting! Should be ok now